exercises/practice/protein-translation/.meta/Example.roc

```diff
@@ -0,0 +1,40 @@
+module [toProtein]
+
+Codon : List U8
+AminoAcid : [Cysteine, Leucine, Methionine, Phenylalanine, Serine, Tryptophan, Tyrosine]
+Protein : List AminoAcid
+
+toInstruction : Codon -> Result [Append AminoAcid, Stop] [KeyNotFound]
+toInstruction = \codon ->
+    Dict.fromList [
+        (['A', 'U', 'G'], Append Methionine),
+        (['U', 'U', 'U'], Append Phenylalanine),
+        (['U', 'U', 'C'], Append Phenylalanine),
+        (['U', 'U', 'A'], Append Leucine),
+        (['U', 'U', 'G'], Append Leucine),
+        (['U', 'C', 'U'], Append Serine),
+        (['U', 'C', 'C'], Append Serine),
+        (['U', 'C', 'A'], Append Serine),
+        (['U', 'C', 'G'], Append Serine),
+        (['U', 'A', 'U'], Append Tyrosine),
+        (['U', 'A', 'C'], Append Tyrosine),
+        (['U', 'G', 'U'], Append Cysteine),
+        (['U', 'G', 'C'], Append Cysteine),
+        (['U', 'G', 'G'], Append Tryptophan),
+        (['U', 'A', 'A'], Stop),
+        (['U', 'A', 'G'], Stop),
+        (['U', 'G', 'A'], Stop),
+    ]
+    |> Dict.get codon
```

Comment:
I suggest using pattern matching here rather than allocating an intermediate dictionary.

Reply:
Ah of course, thanks